routes/course_memberships.js

```diff
@@ -0,0 +1,15 @@
+const router = require('express').Router({ mergeParams: true });
```

Comment:
?
Is course memberships routes nested inside course routes?

Reply:
If yes, there is no need to specify semester_name and module_code in the routes below

Reply:
if no, please remove the `mergeParams` here